core/src/test/java/org/apache/calcite/test/SqlJsonFunctionsTest.java

```diff
@@ -55,6 +55,29 @@
  * Unit test for the methods in {@link SqlFunctions} that implement JSON processing functions.
  */
 public class SqlJsonFunctionsTest {
+  public static final String INVOC_DESC_JSON_VALUE_EXPRESSION =
```

Comment:
Same issue. We have already removed these constants in https://github.com/apache/calcite/commit/6c9557fb554cd9e7178987030681982f2e85308e.